test/e2e/framework/pv_util.go

```diff
@@ -92,124 +91,181 @@ type PersistentVolumeClaimConfig struct {
 }
 
 // Clean up a pv and pvc in a single pv/pvc test case.
-func PVPVCCleanup(c clientset.Interface, ns string, pv *v1.PersistentVolume, pvc *v1.PersistentVolumeClaim) {
-	DeletePersistentVolumeClaim(c, pvc.Name, ns)
-	DeletePersistentVolume(c, pv.Name)
+// Note: delete errors are appended to []error so that we can attempt to delete both the pvc and pv.
+func PVPVCCleanup(c clientset.Interface, ns string, pv *v1.PersistentVolume, pvc *v1.PersistentVolumeClaim) []error {
+
+	var errs []error
+	var err error
+
+	if pvc != nil {
+		if err = DeletePersistentVolumeClaim(c, pvc.Name, ns); err != nil {
+			errs = append(errs, fmt.Errorf("Failed to delete PVC %q. Error: %v", pvc.Name, err))
```

Comment:
There are approx 383 examples of error messages containing "Failed" and ~102 examples of the word "Error" in error messages. So I am not sure if all lowercase is a convention, at least for the e2e tests.